clientcontroller/babylon/consumer.go

```diff
@@ -463,3 +463,54 @@ func (bc *BabylonConsumerController) UnjailFinalityProvider(fpPk *btcec.PublicKe
 
 	return &types.TxResponse{TxHash: res.TxHash, Events: res.Events}, nil
 }
+
+// QueryPublicRandCommitList returns the public randomness commitments list from the startHeight to the last commit
+// the returned commits are ordered in the accenting order of the start height
+func (bc *BabylonConsumerController) QueryPublicRandCommitList(fpPk *btcec.PublicKey, startHeight uint64) ([]*types.PubRandCommit, error) {
+	fpBtcPk := bbntypes.NewBIP340PubKeyFromBTCPK(fpPk)
+
+	pagination := &sdkquery.PageRequest{
+		Limit:   1,
```

Comment:
why limit 1?

Reply:
We can remove the limit but I thought with `1` it's cleaner as we don't need to iterator the commit map and sort them by the start height